assignments/callbacks.js

```diff
@@ -21,22 +21,42 @@ const items = ['Pencil', 'Notebook', 'yo-yo', 'Gum'];
 
 */
 
-
-function getLength(arr, cb) {
-  // getLength passes the length of the array into the callback.
+// getLength passes the length of the array into the callback.
+ 
+function getLength(arr, cb) {  return cb(arr.length);
```

Comment:
This is good, just make sure to put return on the next line
solution:
function getLength(arr, cb) {
//   // getLength passes the length of the array into the callback.
//   console.log(cb(arr));
// }

// let arrayLength = function(items) {
//   return items.length;
// }

// getLength(items,arrayLength);